lib/route-on-event.js

```diff
@@ -8,30 +8,32 @@ exports.routeFromLocation = function routeFromLocation (event, router) {
 }
 
 exports.routeFromLinkClick = function routeFromLinkClick (event, router) {
-  // ignore if it could open a new window, if a right click
-  if (
-    event.metaKey || event.shiftKey || event.ctrlKey || event.altKey ||
-    event.which === 3 || event.button === 2
-  ) return
+  setTimeout(() => {
```

Comment:
arrow functions are not supported in all supported browsers, and this code is not transformed using babel or any other tool.